source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/075_DML_PIMS_ROLE_CLAIM.sql

```diff
@@ -102,26 +102,12 @@ SELECT @activityAdd = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'activity-add';
 SELECT @activityEdit = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'activity-edit';
 SELECT @activityDelete = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'activity-delete';
 
-DECLARE @projectView BIGINT;
```

Comment:
@dfilteau sorry, should have been more explicit. You fixed the issue with PIMS_CLAIM but this file needs to be corrected as well.